Dockerfile

```diff
@@ -2,12 +2,10 @@ FROM php:5.6-apache
 
 # docker php image has its own way of installing a module
 RUN apt-get update \
- && apt-get install --no-install-recommends -y libpq-dev=9.6.23-0+deb9u1 \
+ && apt-get install --no-install-recommends -y libpq-dev=9.6.24-0+deb9u1 \
```

Comment:
9.6 is end of life so we should look to upgrade it probably

Reply:
Isn't the version bound to the managed postgres instance used as backend?

Reply:
This is the client which should match the server if it doesn’t then that’s a different problem…